requirements/software-metadata-requirements.md

```diff
@@ -0,0 +1,187 @@
+# CLARIAH Software Metadata Requirements and Instructions
+
+## Introduction
+
+This document specifies requirements and instructions for software metadata for
+CLARIAH software and services. It is aimed at the developers of software and
+attempts to provide clear documentation.
+
+The main principle underlying the way we keep our software metadata is that
+**metadata is kept at the source and by the source of the software**. This
+means that the tool developers themselves are responsible for providing
+accurate metadata and that the best place to keep this metadata is *with the
+source code* of the software, in a version-controlled repository.
+
+This is in line with current best-practices as most programming language
```

Comment:
The approach aligns with current practices (using package definitions local to the programming language used) but which _best_ practices?